apps/wallet-connect/src/App.tsx

```diff
@@ -197,14 +139,14 @@ const App = () => {
     <>
       {/* WalletConnect */}
       {wcClientData && (
-        <StyledParagraph size="sm">
+        <StyledWrapper>
```

Comment:
This change is to remove the warning in the console:
![Captura de pantalla 2022-01-18 a las 17 47 59](https://user-images.githubusercontent.com/26763673/150131305-b0f87220-b9b4-4516-80cc-5074b185e5e0.png)